aip/general/0151.md

```diff
@@ -108,6 +108,11 @@ Errors that occur over the course of an operation **may** be placed in the
 metadata message. The errors themselves **must** still be represented with a
 [google.rpc.Status][] object.
 
+### Backwards compatibility
+
+Changing either the `response_type` or `metadata_type` of a long-running operation
```

Comment:
Nit: Might be over kill, but a little subheading might be nice for the anchor it provides when referring to this point in critique, a tool finding/code, or email.

Reply:
Isn't a subheading for "Backwards compatibility" sufficient? (I think we'll get an anchor for that automatically.)

Reply:
Yes, you are right. In my skimming, I thought this change was in AIP-180, but we are in the relevant AIP! Nevermind.